src/app/profile/[id]/layout.tsx

```diff
@@ -0,0 +1,24 @@
+'use client'
+
+import React from 'react'
+import useMedia from '@/hook/useMedia'
+import { Container, Stack, Box } from '@mui/material'
+import * as style from '../../my-page/layout.style'
+
+const Layout = ({ children }: { children: React.ReactNode }) => {
+  const { isPc } = useMedia()
+  return (
+    <Container sx={style.container}>
+      <Stack
+        justifyContent={'center'}
+        direction={isPc ? 'row' : 'column'}
```

Comment:
direction등등의 스타일 관련 프로퍼티들은 브레이크 포인트가 적용됩니다. 다음과 같이 수정하면 layout을 클라이언트 컴포넌트로 바꾸지 않고도 반응형 디자인을 적용할 수 있습니다. 참고해주세요!
```suggestion
        direction={['column', 'row']}
```